alacritty.yml

```diff
@@ -429,13 +429,13 @@
 
 # Shell
 #
-# You can set `shell.program` to the path of your favorite shell, e.g.
+# You can set `shell.program` to the path of your favorite shell, e.g.,
```

Comment:
Why did you put commas after all the `e.g.`s in this file? That's not correct…

Reply:
I looked into it more and I think it's an American English vs. British English thing... I'll revert if you want to follow British English style guides, but there aren't British spellings everywhere (like "favourite" on this line), so I assumed American English would be preferred. https://jakubmarian.com/comma-after-i-e-and-e-g/

Reply:
Doesn't matter if it's british or american, no reason to change something that is correct. You're just introducing unnecessary changes.

Reply:
Ok. To American readers it is wrong to omit the comma, which is why I changed it initially, but I'll take them out.